joomla/joomla-cms
# [4.0] "Authentification" correction

Correction of language and hint positioning for two factor authentication field on login page.

Pull Request does not have a related issue - I identified issue while checking through translations, then found an issue where it appears in the site.

### Summary of Changes
- Correct "Authentification" to "Authentication" in COM_LOGIN_TWOFACTOR string (Authentification is valid, but not proper en_GB)
- On viewing where the string was being used, I then found the layout is incorrect, so have fixed that also.
- scss update to create new class for .form-control-hint to fix the layout.

### Testing Instructions
- In Joomla 4, enable one of the two factor authentication plugins
- Log out
- On the log in screen you'll now see the Secret Key box, with the spelling issue and the incorrect formatting of the hint that uses COM_LOGIN_TWOFACTOR string.
- Apply PR
- Log out again and check that the Log in screen now has the corrected en_GB spelling as well as better formatting per the AFTER result below.

### Actual result BEFORE applying this Pull Request
![Before-autheitification](https://user-images.githubusercontent.com/5515866/96845857-e25afe80-149c-11eb-977c-a618df0238e1.jpg)
- Spelling error still visible - COM_LOGIN_TWOFACTOR string will display as "For Two-Factor Authentification" for en_GB
- Hint for Secret Key is supposed to be positioned on the right, but due to double span the text-right class fails to do so. 

### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/5515866/96849525-6f07bb80-14a1-11eb-8210-4123ffb63a76.png)
- COM_LOGIN_TWOFACTOR string will now show as "For Two-Factor Authentication"
- Hint for Secret Key will now be displayed above the field, to the right of the field.

### Documentation Changes Required
No. en_GB translation will be updated, but will most likely have been correctly translated into other languages previously. I just spotted the issue when translating en_US.
